BlackNectar/BlackNectar/FilterViewController.swift

```diff
@@ -216,16 +263,30 @@ extension FilterViewController {
     
 }
 
-//MARK: Loads Stores When User Pans
+//MARK: Style Menu Code
 extension FilterViewController {
     
-    func mapView(_ mapView: MKMapView, regionDidChangeAnimated animated: Bool) {
+    func styleMenu() {
         
-        let center = mapView.centerCoordinate
+        farmersMarketsButton.layer.borderColor = UIColor.white.cgColor
+        farmersMarketsButton.layer.borderWidth = 1
+        farmersMarketsButton.layer.cornerRadius = 7
```

Comment:
This is the sort of thing that can be done from Storyboard by using `@IBDesignable`.
It is recommended to do visual stuff in the visual editor, to maximize productivity and minimize code footprint.

Also, the cornerRadius is supposed to be 5. :wink: